swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
+from operator import attrgetter
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else list() #using empty list constructor to explicitly note list data type expected
```

Comment:
I would strongly suggest placing comments directly above the code they relate to, rather than on the same line. Unless a comment is very short, placing it on the same line will quickly make the line very long, making it harder to read by needing to scroll, or making it harder to see the indentation at a glance by wrapping to another line.
```suggestion
    def __init__(self, inventory=None):
        # using empty list constructor to explicitly note list data type expected
        self.inventory = inventory if inventory is not None else list() 
```